forem/forem
# ✂️✂️✂️ Remove API v0 support ✂️✂️✂️

## What type of PR is this? (check all applicable)


## Description

After the transition period and now that the API V1 docs are ready we're in shape to get rid of the old V0 API endpoints and stick with the new major version going forward.

This PR gets rid of the concerns extraction of logic (used to avoid repeating the code on shared controllers) so it will all be easier to find/understand in the codebase.

## Related Tickets & Documents

N/A

## QA Instructions, Screenshots, Recordings

- Check out the code locally and try out some API endpoints
- Check out the specs in place and ensure they're ✅ 

### UI accessibility concerns?

None - backend change

## Added/updated tests?


## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

This is a sensitive change. I strongly recommend some manual monitoring over the next hour or two after this is merged to ensure no edge case scenarios are causing an elevated amount of user facing errors

## [optional] What gif best describes this PR or how it makes you feel?

![monkey pulling bunny out of brain "tada!"](https://media.giphy.com/media/xTiTnFClK0PdoDeJTW/giphy.gif)
